File: data/data.code.java.server/ReqTestJsOk.java
package protocol;


import network.Packet;
import network.PacketUtil;

import java.util.List;
import java.util.ArrayList;


public class ReqTestJsOk
{
	private long u64;
	private long i64;


	public ReqTestJsOk(Packet packet)
	{
		u64 = packet.readLong();
		i64 = packet.readLong();
	}



	public long getU64()
	{
		return this.u64;
	}

	public long getI64()
	{
		return this.i64;
	}

}

File: data/data.code.csharp.server/ReqTestJsOk.cs
using System.Collections;
using System.Collections.Generic;


public class ReqTestJsOk
{
	private ulong _u64;
	private long _i64;


	public ReqTestJsOk(Packet packet)
	{
		this._u64 = packet.ReadUlong();
		this._i64 = packet.ReadLong();
	}


	public ulong u64
	{
		get { return this._u64; }
		set { this._u64 = value; }
	}

	public long i64
	{
		get { return this._i64; }
		set { this._i64 = value; }
	}

}
